ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-main__edge.yaml

```diff
@@ -0,0 +1,99 @@
+base_images:
```

Comment:
this is rather a general question: why do we move existing configuration to ``edge`` variant?
is it to be able to copy a very specific configuration when branching-out to the ``release-x.y`` branches?

Reply:
The discussion was about using the canonical jobs for shipping the images into the ocp release payload, in a similar manner to the other openshift components that are part it (there is also a technical motivation, as it's not possible to specify multiple promotion targets in the same job). Moving to exisitng jobs to the `edge` variant will preseve the current behavior, as their artifacts will be continued to be promoted in their own namespaces